playground/TestPlatform.Playground/Program.cs

```diff
@@ -335,4 +335,4 @@ public void HandleStopTestSessionComplete(StopTestSessionCompleteEventArgs? even
     {
 
     }
-}
+}
```

Comment:
I would keep the new line at the end to reduce git conflicts.

Reply:
Fixed.